examples/dind.groovy

```diff
@@ -7,34 +7,34 @@ docker.sock does not need to be mounted as in dood.groovy.
 May or may not work depending on cluster policy: https://kubernetes.io/docs/concepts/policy/pod-security-policy/
 */
 podTemplate(yaml: '''
-apiVersion: v1
-kind: Pod
-spec:
-  volumes:
-  - name: docker-socket
-    emptyDir: {}
-  containers:
-  - name: docker
-    image: docker:19.03.1
-    command:
-    - sleep
-    args:
-    - 99d
-    volumeMounts:
-    - name: docker-socket
-      mountPath: /var/run
-  - name: docker-daemon
-    image: docker:19.03.1-dind
-    securityContext:
-      privileged: true
-    volumeMounts:
-    - name: docker-socket
-      mountPath: /var/run
-''') {
-    node(POD_LABEL) {
-        writeFile file: 'Dockerfile', text: 'FROM scratch'
-        container('docker') {
-            sh 'docker version && DOCKER_BUILDKIT=1 docker build --progress plain -t testing .'
-        }
+              apiVersion: v1
```

Comment:
I do not find this helpful. Easier to copy & paste actual YAML from another editor if it is unindented.

Reply:
cf. https://github.com/jenkinsci/kubernetes-plugin/pull/942 . I don't have a strong opinion about this.